gen3/bin/kube-setup-pidgin.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Deploy pidgin into existing commons
+# This fragment is pasted into kube-services.sh by kube.tf.
+#
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
```

Comment:
Add a call to `kube-setup-pidgin` to `kube-setup-all`

Reply:
done